KitX Website Flutter/kitx_website/lib/utils/translation/en_us.dart

```diff
@@ -0,0 +1,28 @@
+﻿// ignore: constant_identifier_names
+const Map<String, String> en_US = {
+  'Public_Docs': 'Docs',
+  'Home_Page': 'Home',
+  'Plugins_Page': 'Plugins',
+  'Docs_Page': 'Docs',
+  'Milestones_Page': 'Milestones',
+  'Settings_Page': 'Settings',
+  'Download_Tested': 'Tested on @content',
+  'Download_Supported': '@content supported',
+  'Download_NoTest': 'No test yet',
+  'Download_ApplyTo': 'Apply to @content',
+  'Download_ElderVersions': 'Elder Versions (KitX 1/2)',
+  'Download_LookingForElderVersions': 'Looking for elder versions ? (Not recommend)',
+  'Download_SelectDownloadSource': 'Select download source: ',
+  'Bits_32': '32 bits',
+  'Bits_64': '64 bits',
+  'CPU_Bits_32': '32 bits CPU',
+  'CPU_Bits_64': '64 bits CPU',
+  'CPU_x86': 'x86 CPU',
+  'CPU_x86_64': 'x86 CPU 64 bits',
```

Comment:
Usually called `x86_64 CPU` or `x64 CPU`.
But I think `x86_64 Architecture` is better.

Reply:
x86 is mean to architecture, and x64 is mean to bits width, so I still think we should not mix x86 and x64
So I think `x86 64 bits` or `x86 32 bits` is better than `x86 cpu` or `x64 cpu`